### ISSUE TITLE
convert a tensor to matrix

### ISSUE THREAD

**Issue by jeansely** [tensorflow/tensorflow#9695]
Hi,
 output of first layer cnn , "if  input was a image with size[100,150] then the output layer is a tensor (1,100,150,32)"
for slicing The first image from all 32 output images, i used this code:
conv1 = tf.layers.conv2d( image, filters=32, kernel_size=[5, 5], padding="same", activation=tf.nn.relu)
first_image=slice(conv1,[0,0,0,0],[1, 100, 152, 1])

>>conv1 is  tensor (1, 100, 150 ,32)
>>first_image is tensor (1, 100 ,150 , 1)

has the Tensorflow  a command for convert a tensor to 2-D matrix directly ?, or how can convert a tensor to matrix in Tensorflow?

**Comment by drpngx** [comment]
You can reshape a tensor using `tf.reshape`, if that's what you need.

There might be a function here: https://www.tensorflow.org/api_guides/python/array_ops#Shapes_and_Shaping that might help you.

This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
